examples/utils/example-utils/src/migrationTool/sameContainerMigrationTool.ts

```diff
@@ -322,10 +326,14 @@ export class SameContainerMigrationTool extends DataObject implements ISameConta
 				return;
 			}
 
-			const watchForAccepted = (key: string) => {
+			const watchForAccepted = (key: string, sequenceNumber: number) => {
 				if (key === newVersionKey) {
 					this.pactMap.off("accepted", watchForAccepted);
-					console.log("Resolving this._acceptedP: Saw acceptance during run time");
+					this._acceptedSeqNum = sequenceNumber;
```

Comment:
Related to comment above about programmatic inspection - here this._acceptedSeqNum won't get defined if this.pactMap.get() finds a value in the logic above on L323.